src/main/java/com/iridium/iridiumskyblock/listeners/PlayerMoveListener.java

```diff
@@ -17,6 +18,10 @@
 
     @EventHandler
     public void onPlayerMove(PlayerMoveEvent event) {
+        if (!IridiumSkyblockAPI.getInstance().isIslandWorld(event.getFrom().getWorld()) && event.getTo() != null && !IridiumSkyblockAPI.getInstance().isIslandWorld(event.getTo().getWorld())) {
```

Comment:
This is bad, even tho they arnt in the skyblock world, still means they should have the teleport dont move thingy also IslandManager#getIslandViaLocation includes the world checks